core/templates/domain/objects/UnitsObjectFactory.ts

```diff
@@ -254,11 +252,7 @@ export class UnitsObjectFactory {
 
     var compatibleUnits = this.toMathjsCompatibleString(units);
     if (compatibleUnits !== '') {
-      try {
-        unit(compatibleUnits);
-      } catch (err) {
-        throw new Error(err);
-      }
+      unit(compatibleUnits);
```

Comment:
Why was the try-catch block removed?

Reply:
Because it is not needed it just encloses `SyntaxError` thrown from `unit`.